lib/Mojo/WebSocketProxy/Backend/ConsumerGroups.pm

```diff
@@ -174,17 +184,20 @@ Returns undef.
 sub call_rpc {
     my ($self, $c, $req_storage) = @_;
 
-    my $rpc_response_cb = $self->get_rpc_response_cb($c, $req_storage);
+    my $rpc_response_cb               = $self->get_rpc_response_cb($c, $req_storage);
     my $before_get_rpc_response_hooks = delete($req_storage->{before_get_rpc_response}) || [];
-    my $after_got_rpc_response_hooks  = delete($req_storage->{after_got_rpc_response})  || [];
-    my $before_call_hooks             = delete($req_storage->{before_call})             || [];
+    my $after_got_rpc_response_hooks  = delete($req_storage->{after_got_rpc_response}) || [];
+    my $before_call_hooks             = delete($req_storage->{before_call}) || [];
     my $rpc_failure_cb                = delete($req_storage->{rpc_failure_cb});
+    my $category_name                 = $req_storage->{msg_group} || 'general';
```

Comment:
Why are we not deleting this one too? (or perhaps the question is: why are we deleting those keys?

Reply:
Those key/values will be used to construct the `request` we are sending to rpc, so probably those other keys are not required and should be deleted, but as I didn't know if `msg_group` is being used in layers below, I didn't take the risk of deleting it. Would be nice if @kirill-binary could suggest if I can delete `msg_group` key as well or not.

Reply:
I'm not sure the reason why we `delete` callbacks i tryed to be consistent with HTTP transport.
https://github.com/binary-com/perl-Mojo-WebSocketProxy/blob/master/lib/Mojo/WebSocketProxy/Backend/JSONRPC.pm#L91-L94

IMHO `msg_group` we can keep. In theory it could be useful in hooks.